src/tests/ftest/ior/intercept_verify_data_integrity.py

```diff
@@ -42,17 +59,60 @@ def test_ior_intercept_verify_data(self):
         self.add_pool()
         self.add_container(self.pool)
 
-        intercept = os.path.join(self.prefix, 'lib64', 'libioil.so')
-        results = dict()
-        client_count = len(self.hostlist_clients)
-        w_clients = self.hostlist_clients[0:client_count - 1]
-        wo_clients = [self.hostlist_clients[-1]]
+        # Start dfuse for POSIX api. This is specific to interception library test requirements.
+        self.start_dfuse(self.hostlist_clients, self.pool, self.container)
+
+        # Setup the thread manager
+        thread_manager = ThreadManager(run_ior, self.timeout - 30)
+        index_clients_intercept = [
+            (0, self.hostlist_clients[0:-1], os.path.join(self.prefix, 'lib64', 'libioil.so')),
+            (1, self.hostlist_clients[-1:], None),
+        ]
+        self.job_manager = []
+        for index, clients, intercept in index_clients_intercept:
+            # Add a job manager for each ior command
+            job_manager = get_job_manager(self, "Mpirun", None, False, "mpich", self.timeout - 35)
+
+            # Create a unique ior test file for each thread
+            test_file_items = ["testfile", str(index)]
+            if intercept:
+                test_file_items.append("intercept")
+            test_file = os.path.join(self.dfuse.mount_dir.value, "_".join(test_file_items))
```

Comment:
Maybe replace `index` above in `index_clients_intercept` with `testfile_0` and `testfile_1_intercept` respectively? Mostly for debugging purposes to easily identify which file belongs to which config

Reply:
Fixed